src/components/RecipesDisplay/RecipeBrief.css

```diff
@@ -0,0 +1,82 @@
+.recipe-brief-container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  margin: 1.25em;  /* 20px */
+
+  font-family: Poppins;
+}
+
+.recipe-card{
+  width: 300px;
+  height: 225px;
+  margin-bottom: 1.94em; /* 31px */
+  border-radius: 24px;
+  background-position: center;
+  background:   linear-gradient(180deg, lightgray 50%, #000 73.77%), no-repeat center / cover;
+
+  transition: transform 0.5s ease, opacity 0.5s ease, box-shadow 0.5s ease; 
+
+  cursor: pointer;
+  margin-bottom: 1.94em; /* 31px */
+
+  /* shadow elevation 1 */
+  box-shadow: 0px 0px 30px 0px rgba(21, 26, 30, 0.06); 
+}
+
+.recipe-name{
+  padding: 0px 1.06em 0.69em; /* 0px 17px 11px */
+
+  font-size: 1.31rem; /* 21px */
+  font-weight: 600;
+  color: #FFF;
+  
+  padding: 0px 1.06em 0.69em; /* 0px 17px 11px */
+  font-size: 1.31rem; /* 21px */
+}
+
+/* Hover Effect */
+
+.recipe-card:hover{
+  width: 350px;
+  height: 225px;
+}
+
+/* Media Queries */
+
+@media (max-width: 300px) {
+  .recipe-card{
+    width: 225px;
+    height: 166px;   
+    
+  }
+
+  .recipe-name{
+    font-size: 1rem; /* 16px */
+  }
+
+  .recipe-card:hover{
```

Comment:
you should nest hover in recipe-card. 

.recipe-card {
   ....
    &:hover{
        ....
    }
}

Reply:
update the style as recommended.